### ISSUE TITLE
Baseflow Diagrams from Model Runs

### ISSUE THREAD

**Issue by glenncampagna** [HARPgroup/HARParchive#248]

**Comment by glenncampagna** [comment]
- 4 new graphs created using R for AGWS were uploaded to directory /media/model/p6/out/land/hsp2_2022/eos/ , also accessed at http://deq1.bse.vt.edu:81/p6/out/land/hsp2_2022/eos/ 
- Each graph represents chart type 1 from above but uses a different time interval for the AGWS values (hourly, daily, weekly, and monthly)
- R was used for easier data and time series manipulation compared to excel while we are determining the timestep we will use in the future

**Comment by rburghol** [comment]
Great stuff! These examples look like there may be a trend towards lower GW storage over time... interesting. 

Note: we absolutely want to continue using R - not excel :).

One request: paste the sample images in the issue body or an issue comment, with descriptions. That way anyone viewing the issue doesn't need to jump around to see the examples.

**Comment by juliabruneau** [comment]
### AGWS Graphs
by @nicoledarling and @glenncampagna 

![image](https://user-images.githubusercontent.com/104520318/177548542-458bd967-ba94-4d82-908f-13cf7caa08c0.png)

![image](https://user-images.githubusercontent.com/104520318/177548610-ee22ec80-c598-4464-9ae3-e1669c1bc456.png)

![image](https://user-images.githubusercontent.com/104520318/177548652-24dbf465-5aba-435e-b57a-cbb89470e93a.png)

![image](https://user-images.githubusercontent.com/104520318/177548707-fbf9b848-7ac8-4cf3-abfb-26f32fa05914.png)

**Comment by rburghol** [comment]
Thanks @juliabruneau ! These are thought provoking.

**Comment by juliabruneau** [comment]
### GWS Graphs

The yearly active groundwater, lower zone, and upper zone storage is represented in the graph below from 1984 until 2020. 

It was easy to create a stacked graph using Excel, but we'll create one using **R** as well and post the code in this comment.

![image](https://user-images.githubusercontent.com/104520318/177550239-254d36a1-d530-4a7f-920c-b2deb3638098.png)

**Comment by rburghol** [comment]
This is also very thought provoking. The tiny amount of AGWS is in stark contrast to both of the higher zones. I suppose it *could* be reasonable that since uzs and lzs represent water in soil pores and AGWS is in rock, that AGWS should be lesser, or at least more transient.  But still... big difference (in this one single case study).

At minimum this tells me we need to take a quantitative measure, like mean or median storage during the simulation so we can do quantitative analysis.

**Comment by rburghol** [comment]
Hey team - I just added another wish to the list (in the top of the issue body) -- I am wondering if the h5 posesses enough info to allow us to determine the base flow (AGWO) into a river reach?  That would be more efficient that getting the baseflow **out** of all 35 land uses, summing them up, then analyzing it for a reach.

**Comment by rburghol** [comment]
@juliabruneau I think for starters, we visualize what IS there.  Inches per interval tells us something that is summable (inches per day, month, year), and plottable. 

I agree that 0.005 seems small, but, if this is an hourly value, then 24*0.005 = 0.12 inches per day, and that is *real* water, since 365 * 0.1 = 36.5 inches/year, which we now know is far more than flows out of our streams, since > 75% of the 40 inches/year of rainfall is evaporated away.

**Comment by juliabruneau** [comment]
> @juliabruneau I think for starters, we visualize what IS there. Inches per interval tells us something that is summable (inches per day, month, year), and plottable.
> 
> I agree that 0.005 seems small, but, if this is an hourly value, then 24*0.005 = 0.12 inches per day, and that is _real_ water, since 365 * 0.1 = 36.5 inches/year, which we now know is far more than flows out of our streams, since > 75% of the 40 inches/year of rainfall is evaporated away.

You are correct. Instead of using the average, for the in/interval unit I had to sum the daily values together in order to get the right values. Here is a graph I plotted just now, and I'm working on different time periods as well.

![image](https://user-images.githubusercontent.com/104520318/177594816-3ca04dee-8b87-406e-bee5-08d774d2bf79.png)

**Comment by megpritch** [comment]
For item 3 ("A plot of baseflow OUT of AGWO, that somehow indicates summer...") Glenn and I came up with this plot: 
![image](https://user-images.githubusercontent.com/104520247/177625618-9deca621-9714-4173-bc78-aa00aac261e2.png)

**Comment by glenncampagna** [comment]
@rburghol , here is my rough draft for plot type 2 for the whole time period
![image](https://user-images.githubusercontent.com/104520197/177638118-d870f53e-d7bb-42e9-b1ce-31a4ed7c4c99.png)
I'm working on adding a legend, but the blue corresponds to LZS, the red to UZS, and the green to AGWS

**Comment by glenncampagna** [comment]
Progress:
- We believe we've found values for l90_Runit and l90_agwo_Runit from A51800
l90_Runit = 0.009517 cfs/sq mi
l90_agwo_Runit = 0.004750 cfs/sq mi

@rburghol when you tell us to 'export' these values, we think you mean adding them to VaHydro under the appropriate land segment feature and model scenario which we reviewed yesterday. However we are unsure of how exactly to use the script from yesterday to upload our values where we want them to go. We are referencing the diagram you provided us but can't quite figure out what all needs to be listed in the command (varkey, propname, prop value, etc.)

**Comment by glenncampagna** [comment]
### Plot 3 for the whole time period in units of cfs/sq mi
![image](https://user-images.githubusercontent.com/104520197/178051801-c42ee409-edae-45a2-9483-a9f21e6fa7bf.png)

**Comment by rburghol** [comment]
Thanks @glenncampagna - we will review this process today at the meeting!
